OperatingSystem-Services/Service-SOC/BasicDocumentation/Wazuh-DiskSpaceError.md

```diff
@@ -0,0 +1,33 @@
+# LVM Out of Space
```

Comment:
Rather than the error name which is slightly distinct from the file name, Maybe change the file name.